pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -268,7 +291,27 @@ func (configgen *ConfigGeneratorImpl) buildSidecarListeners(env *model.Environme
 						Filters: []listener.Filter{filter},
 					},
 				},
-			})
+			}
+			mutable := &plugin.MutableObjects{
+				Listener:     ipTablesListener,
+				FilterChains: make([]plugin.FilterChain, len(ipTablesListener.FilterChains)),
+			}
+			mp := mixerplugin.NewPlugin()
```

Comment:
? This is a direct usage of a plugin in the main code. Why dont you follow the same structure as all the other plugin usage points and have logic to skip processing this listener in the plugins?